pytorch_lightning/metrics/functional/image_gradients.py

```diff
@@ -0,0 +1,80 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional, Sequence, Tuple
+
+import torch
+import torch.nn.functional as F
+
+from pytorch_lightning.metrics.utils import _check_same_shape
+
+
+def _image_gradients_update(img: torch.Tensor, device: torch.device) -> torch.Tensor:
```

Comment:
```suggestion
def _image_gradients_validate(img: torch.Tensor, device: torch.device) -> torch.Tensor:
```

Reply:
made the recommended changes in the latest commit